### ISSUE TITLE
Menu item groups are not narrated on HTML page

### ISSUE THREAD

**Issue by adamsamec** [FreedomScientific/VFO-standards-support#581]
### Summary
On an HTML page, if a menu contains grouped items, when user navigates from an item in one group to an item in another group, JAWS should announce the label of the entered group. Here, by "menu" an element with role="menu" is understood, and this element contains elements with role="menuitem" which are contained in elements with role="group" and aria-label="&lt;group name&gt;".

### Repro steps
1. Turn on JAWS.
2. Open this example: https://1j2rm.csb.app/
3. Using the Tab key, navigate to the "Events" button and activate it to enter the menu.
4. Press the Down arrow key several times to go through the entire menu and observe the missing narration  of menu item groups named "Current events" and "Events next year" as you navigate from one group to another.

### Expected result
When navigated from one group of menu items to another, e.g. from "Current events" to "Events next year", the entered group label should be announced.

### Actual result
JAWS announces only the label of the menu item which the user navigated to, not the entered group label. The expected result can be observed when using the NVDA screen reader for instance for the abovementioned steps.

#### JAWS version and build number
JAWS 2021.2107.12 ILM
 
#### Operating System and version
Windows 10 Enterprise
Version 21H1 (OS Build 19043.1237)

#### Browser and version:
Google Chrome 94
